### DISCUSSION TITLE
[Tutorial] How to run  autofishbot on android

### DISCUSSION THREAD

**Discussion by Yurkhun** [thejoabo/virtualfisher-bot#68] [Show and tell]
### Hi guys,
**It's a simple video showing how to run this python script [AutoFishBot] on android using Termux..**

steps:
**1- [https://f-droid.org/en/packages/com.termux/](https://f-droid.org/en/packages/com.termux/)**
_You can download termux from google play too but normally I get an error when trying to update/upgrade when using google play version..._
**2- apt update**
**3- apt upgrade**
**4- pkg install python**
**5- pkg install mc**
**6- termux-setup-storage**
**7- remove "windows-curses==2.3.1" in requirements file** 
**8- mc**
**9- python -m pip install -r requirements.txt**
**10- python autofishbot.py config_name**

https://user-images.githubusercontent.com/95500637/209002551-2fc85f54-18ff-4b4f-8212-d3a87976979f.mp4

**Comment by Yurkhun** [comment]
Video is 2x speed because the file was too heavy to upload to github..
if you want to watch the video in normal speed and quality. 
I Uploaded  it to youtube: [https://www.youtube.com/watch?v=QcRP5ay2q0g](https://www.youtube.com/watch?v=QcRP5ay2q0g)

**Comment by thejoabo** [comment]
@Yurkhun Excellent tutorial, thank you very much for your effort ! I didn't even know it was possible to run it on android !

**Reply by Yurkhun** [reply]
Thanks for the compliment ☺️  I knew about termux for a long time, fortunately it work very well to run python 😁

**Comment by Yurkhun** [comment]
This is how to open multiple instances in termux.

Push from the left and click in "new session" and then repeat the command to start it.

**1->MC**
**2-> find the folder where the bot files is**
_3->write_: **python autofishbot.py**
**4->select which config you want to run**


https://user-images.githubusercontent.com/95500637/209077428-dbff4185-6855-4187-b5ab-f897e0f20c66.mp4

**Comment by Yurkhun** [comment]
Just for curiosity. 
Termux running in background for 3 hours consumed 1,7% of my battery, so the usage is very low in total it run for 9h consuming 6,8% (need to consider when installing python and everything it consumed more battery I guess... but idle running 3 hours consuming 1,7% means approximately 0,56%/h) 

My battery capacity is 4300 mAh ![Screenshot_20221222-094541_Device care.jpg](https://user-images.githubusercontent.com/95500637/209095311-c83b9849-5365-4aa9-8dbc-1520dd9eac1f.jpg)
